### PR TITLE
GSE131957 Ref matrices

### PR DESCRIPTION
**Pull request by SidhantPuntambekar**
Creating ref matrices for mouseAtlas with study GSE131957 which deals with Identification of DC1 immuno-regulatory modules that limit antitumor CD8+ T cell immunity in lung cancer lesions. Should solve #32.

### CODE REVIEW

**Comment by SidhantPuntambekar** [comment]
@raysinensis Out of the 18 csv counts files in this record, only eight are raw counts. As a result, I only did reference matrices for these eight. The ref matrices and new mouseAtlas are in this PR.

File: R/musMusculus/GSE124952/GSE124952.r [hunk 1]
```diff
@@ -34,7 +34,7 @@ checkRawCounts(as.matrix(mat_PFC))
 GSE124952Normalized <- NormalizeData(mat_PFC)
 GSE124952Normalized
 
-new_ref_matrix <- average_clusters(mat = mat_PFC, metadata = meta_PFC$CellType, if_log = TRUE)
+new_ref_matrix <- average_clusters(mat = GSE124952Normalized, metadata = meta_PFC$CellType, if_log = TRUE)
 new_ref_matrix_hashed <- average_clusters(mat = mat_PFC, metadata = meta_PFC$CellType, if_log = FALSE)
```
Comment (raysinensis) [inline R/musMusculus/GSE124952/GSE124952.r:38]: If we still use the hashed versions, then if_log also needs to be changed

Comment (SidhantPuntambekar) [reply R/musMusculus/GSE124952/GSE124952.r:38]: Do you mean just change it to `FALSE` on line 37?

**Comment by raysinensis** [comment]
Let's ignore all ADT labeled samples. If you look at them, they have very few genes. This is because this is from Antibody-Derived Tags, meaning antibodies that bind cell surface markers, which also are conjugated to DNA tags that can be read out by sequencing. This is more reflective of surface marker expression, not mRNA, which is good for many experiments, but not our clustifyr goal.

**Pull request merged**